src/ensembl/src/content/app/species/components/species-sidebar/SpeciesSidebar.tsx

```diff
@@ -0,0 +1,175 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useEffect } from 'react';
+import { connect } from 'react-redux';
+import classNames from 'classnames';
+
+import ExternalReference from 'src/shared/components/external-reference/ExternalReference';
+
+import { getActiveGenomeId } from 'src/content/app/species/state/general/speciesGeneralSelectors';
+import { getActiveGenomeSidebarPayload } from 'src/content/app/species/state/sidebar/speciesSidebarSelectors';
+
+import { RootState } from 'src/store';
+import {
+  SpeciesSidebarPayload,
+  fetchSidebarPayload
+} from 'src/content/app/species/state/sidebar/speciesSidebarSlice';
+
+import styles from './SpeciesSidebar.scss';
+
+type Props = {
+  activeGenomeId: string | null;
+  sidebarPayload?: SpeciesSidebarPayload | null;
+  fetchSidebarPayload: () => void;
+};
+
+const SpeciesSidebar = (props: Props) => {
+  useEffect(() => {
+    if (!props.sidebarPayload) {
+      props.fetchSidebarPayload();
+    }
+  }, [props.sidebarPayload, props.activeGenomeId]);
+
+  if (!props.sidebarPayload) {
+    return <div>No data to display</div>;
+  }
+  const payload = props.sidebarPayload;
+
+  type AnnotationEntry = {
+    label: string;
+    value?: string | null;
+    url?: string;
+  };
+  const assemblyDate = new Date(payload.assembly_date);
+  const formattedAssemblyDate = Intl.DateTimeFormat('en-GB', {
+    month: 'short',
+    year: 'numeric'
+  }).format(assemblyDate);
+  const annotationEntries: AnnotationEntry[][] = [
+    [
+      {
+        label: 'Provider',
+        value: payload.annotation_provider.name,
+        url: payload.annotation_provider.url
+      },
+      { label: 'Method', value: payload.annotation_method }
+    ],
+    [
+      { label: 'Assembly date', value: formattedAssemblyDate },
+      { label: 'Gencode version', value: payload.gencode_version }
+    ],
+    [
+      { label: 'Database version', value: payload.database_version },
+      { label: 'Taxonomy ID', value: payload.taxonomy_id }
+    ]
+  ];
+
+  return (
+    <div>
+      <div className={styles.speciesDetails}>
+        {payload.common_name && (
+          <span className={styles.common_name}>{payload.common_name}</span>
+        )}
+        {payload.scientific_name && (
+          <span className={styles.scientific_name}>
+            {payload.scientific_name}
+          </span>
+        )}
+      </div>
+
+      {payload.strain && (
+        <div className={styles.strainDetails}>
+          <span className={styles.strain_type}>{payload.strain.type}</span>
+          <span className={styles.strain_value}>{payload.strain.value}</span>
+        </div>
+      )}
+
+      <div className={styles.sectionHead}>Assembly</div>
+      <div className={styles.assemblyDetails}>
```

Comment:
bit of a strange spacing here:

<img src="https://user-images.githubusercontent.com/6834224/97306324-3deb0a80-1856-11eb-8e37-7439d100a664.png" width="400">

compare it with the design:

<img src="https://user-images.githubusercontent.com/6834224/97306436-6541d780-1856-11eb-8b46-d81d365a3081.png" width="400">

Reply:
Updated the styles to match the design